.travis.yml

```diff
@@ -1,8 +1,6 @@
-language: scala
-scala:
-- 2.11.4
+language: java
 jdk:
-- oraclejdk8
```

Comment:
Let's be sure to note this in the release notes. "No longer support jdk8"

Reply:
Already got another branch going for updating docs for the next release.